### ISSUE TITLE
Need postgresql backup and restore scripts

### ISSUE THREAD

**Issue by chappjc** [decred/dcrdata#435]
The /dev folder should have scripts to backup and restore the PostgreSQL database, with matching data from ~/.dcrdata/data/net

**Comment by orthomind** [comment]
I'm considering looking into this issue.  Overall, I'd be interested in managing various scripts/tasks using https://github.com/go-task/task rather than raw scripts.  If you're happy, and I take this on, I'd look at starting with go-task by creating an entry to backup and restore.

**Comment by chappjc** [comment]
sounds good @orthomind

**Comment by orthomind** [comment]
Is this still needed and of interest?  I may have some time to contribute over the next week or two.

**Comment by chappjc** [comment]
Yes, it is.  It's probably just a few commands, but it's valuable as an example at least.
